apis/go/go.mod

```diff
@@ -3,16 +3,15 @@ module github.com/seldonio/seldon-core/apis/go/v2
 go 1.18
 
 require (
-	google.golang.org/grpc v1.49.0
+	google.golang.org/grpc v1.56.3
 	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.2.0
-	google.golang.org/protobuf v1.28.1
+	google.golang.org/protobuf v1.30.0
```

Comment:
Similarly here, latest protobuf is 1.32.0 : https://pkg.go.dev/google.golang.org/protobuf?tab=versions

Reply:
same as above, will follow dependabot security updates for now.